### DISCUSSION TITLE
The handheld sonar beacon is an indestructible item.

### DISCUSSION THREAD

**Discussion by pavel4win** [FakeFishGames/Barotrauma#14428] [Ideas and suggestions]
It is enough to throw the beacon overboard and the monsters (even abyss ones) stop attacking the ship. They attack the beacon and circle around it, but they can’t do anything with the handheld beacon.

**Comment by BlackMoons** [comment]
Id like to add that handheld sonars on active sonar also suffer from this same problem.

Best demonstrated by tossing one overboard after activating, a far distance away from the ship, then spawning a mudraptor on it. They will continuously attack the beacon or handheld sonar.
